src/app/components/listfolder/listfolder.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="list_block">
+  <ul *ngIf="data.type === folder">
+    <app-folder [data]="data" (ToggleHandler)="changeVisible($event)"></app-folder>
+    <app-file [data]="data" [ngClass]="{mapping:isVisible}"></app-file>
```

Comment:
we are using the following way: `[class.mapping]="isVisible"`. could you please change it?